.ci/scripts/run_tests_ucc_mpi.sh

```diff
@@ -24,99 +24,44 @@ export MASTER_ADDR=${HEAD_NODE}
 
 NP=$(wc --lines "$HOSTFILE" | awk '{print $1}')
 
+mpi_params="-np $NP --hostfile ${HOSTFILE} --map-by node --allow-run-as-root -x PATH -x LD_LIBRARY_PATH --mca opal_common_ucx_opal_mem_hooks 1 --mca plm_rsh_args -p12345 "
```

Comment:
why do we need opal_common_ucx_opal_mem_hooks ?

Reply:
the way we build ompi (from master): it says
[swx-clx01:00208] common_ucx.c:164  Warning: UCX is unable to handle VM_UNMAP event. This may cause performance degradation or data corruption. Pls try adding --mca opal_common_ucx_opal_mem_hooks 1 to mpirun/oshrun command line to resolve this issue.